blocks/subscriptions-block/components/Cart/index.test.jsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import { mount } from "enzyme";
```

Comment:
Tests in our repos can't use enzyme moving forward, due to incompatibility with Node v16. Can you at least remove the references to enzyme (and `mount`). Ideally these should all be converted to use `@testing-library/react`, but there are issues running the tests currently in this repo so that can be a follow-up task.

Reply:
This will apply to all test files.